server/protocols/jmap/pom.xml

```diff
@@ -275,6 +287,11 @@
                     <scope>test</scope>
                 </dependency>
                 <dependency>
+                    <groupId>com.jayway.awaitility</groupId>
+                    <artifactId>awaitility</artifactId>
```

Comment:
Before in the PR you introduce this dependence and specify a version for it, you did not it here ? Why did you need to specify before and not here ?